src/pybind/mgr/cephadm/tests/test_cephadm.py

```diff
@@ -1938,3 +1938,99 @@ def test_host_rm_last_admin(self, cephadm_module: CephadmOrchestrator):
         with with_host(cephadm_module, 'test1', refresh_hosts=False, rm_with_force=True):
             with with_host(cephadm_module, 'test2', refresh_hosts=False, rm_with_force=False):
                 cephadm_module.inventory.add_label('test2', '_admin')
+
+    @pytest.mark.parametrize("facts, settings, expected_value",
+                             [
+                                 # All options are available on all hosts
+                                 (
+                                     {
+                                         "host1":
+                                         {
+                                             "sysctl_options":
+                                             {
+                                                 'opt1':'val1',
```

Comment:
looks like a bunch of related failures in make check. First of which is here
```
cephadm/tests/test_cephadm.py:1951:56: E231 missing whitespace after ':'
cephadm/tests/test_cephadm.py:1952:56: E231 missing whitespace after ':'
cephadm/tests/test_cephadm.py:1959:56: E231 missing whitespace after ':'
cephadm/tests/test_cephadm.py:1960:56: E231 missing whitespace after ':'
cephadm/tests/test_cephadm.py:1973:52: E231 missing whitespace after ':'
cephadm/tests/test_cephadm.py:1974:52: E231 missing whitespace after ':'
cephadm/tests/test_cephadm.py:1981:52: E231 missing whitespace after ':'
cephadm/tests/test_cephadm.py:1982:52: E231 missing whitespace after ':'
cephadm/tests/test_cephadm.py:1983:52: E231 missing whitespace after ':'
cephadm/tests/test_cephadm.py:1986:38: E123 closing bracket does not match indentation of opening bracket's line
cephadm/tests/test_cephadm.py:1989:34: E124 closing bracket does not match visual indentation
cephadm/tests/test_cephadm.py:2004:38: E123 closing bracket does not match indentation of opening bracket's line
cephadm/tests/test_cephadm.py:2022:38: E123 closing bracket does not match indentation of opening bracket's line
cephadm/tests/test_cephadm.py:2025:34: E124 closing bracket does not match visual indentation
```

Reply:
fixed